javascript/javascript1/week3/smartPhoneUsageApp.js

```diff
@@ -0,0 +1,42 @@
+//Smart-Phone Usage app:
+//Adding an activity
+let activities = [];
+const limit = 110;
+
+function addActivity(date, activity, duration)
+{
+  if(!date){
+    let todayDate = new Date();
+    date = todayDate.toLocaleDateString();
+  }
+  return activities.push({date: date, activity: activity, duration: duration})
+}
+
+addActivity('5/17/2021', 'Youtube', 30);
+console.log(activities);
+
+function showStatus(activities){
+  let sumOfDuration = 0;
+  if(activities.length === 0){
+    console.log('Add some activities before calling showStatus');
```

Comment:
You should remember to return here, so the function doesn't keep running.

Reply:
Yes-Done